docs/source/cdk/cdk-deployment.rst

```diff
@@ -0,0 +1,11 @@
+************
+cdk deployment
+************
+
+In command line, run the following command to set up the aws environment:
+
+.. code-block::
+    aws configure
```

Comment:
If you're using a `code block` directive, you need to include a language: 
``` 
.. code-block:: bash

    aws configure
```
Notice the line break between the code block part and the code. The indentation is correct.

However, if you just want a code block without syntax highlighting, you can just use the double colon. In this case: 
```
In the command line, run the following command:: 

    aws configure
```

So either way, you need a line break in between the double colon and the actual code block. With the `code-block` directive, you also need to provide it with a language (here, bash). 

You can see these errors including the ones Greg already pointed out above by following the README.md guide under docs/ to install and run sphinx locally. If you need help with poetry, let me know.